src/modules/Security.rb

```diff
@@ -34,6 +34,7 @@
 
 module Yast
   class SecurityClass < Module
+    DEFAULT_ENCRYPT_METHOD = "sha512".freeze
```

Comment:
NP: As you added a public method to expose it, do you think it should be a `private_constant`?